src/analyses/base.ml

```diff
@@ -2349,16 +2348,16 @@ struct
      * expression e may point to *)
     let invalidate_exp exps =
       let args = collect_invalidate ~deep ~warn:true ask gs st exps in
-      List.map (invalidate_address st) args
+      List.of_seq @@ Seq.map (invalidate_address st)@@ List.to_seq args
     in
     let invalids = invalidate_exp exps in
     let is_fav_addr x =
       List.exists BaseUtil.is_excluded_from_invalidation (AD.to_var_may x)
     in
     let invalids' = List.filter (fun (x,_,_) -> not (is_fav_addr x)) invalids in
     if M.tracing && exps <> [] then (
-      let addrs = List.map (Tuple3.first) invalids' in
-      let vs = List.map (Tuple3.third) invalids' in
+      let addrs = List.of_seq @@ Seq.map (Tuple3.first) @@ List.to_seq invalids' in
+      let vs = List.of_seq @@ Seq.map (Tuple3.third) @@ List.to_seq invalids' in
```

Comment:
Same question as above, as there is only a `List.map` that is replaced in each line.